cpp/examples/fetch_dependencies.cmake

```diff
@@ -11,17 +11,22 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
-set(CPM_DOWNLOAD_VERSION v0.35.3)
+set(CPM_DOWNLOAD_VERSION v0.38.5)
 file(
   DOWNLOAD
   https://github.com/cpm-cmake/CPM.cmake/releases/download/${CPM_DOWNLOAD_VERSION}/get_cpm.cmake
   ${CMAKE_BINARY_DIR}/cmake/get_cpm.cmake
 )
 include(${CMAKE_BINARY_DIR}/cmake/get_cpm.cmake)
 
+# allow using local packages if already installed
+set(CPM_USE_LOCAL_PACKAGES ON)
```

Comment:
Do we need this or is it temporary for local testing? To my knowledge, you shouldn't need this if you have a local package that meets the version requirements in a place that CMake can find it.

Reply:
Yeah not needed. Will clean up soon